tests/market/test_future.py

```diff
@@ -238,3 +249,73 @@ def test_orders_per_slot(future_market):
                             "seller_origin_id": None,
                             "creation_time": datetime_to_string_incl_seconds(time_slot2),
                             "type": "Offer"}]}}
+
+    @staticmethod
+    def test_offers_setter(future_market, offer):
+        """Test reassigning the offers member of the future market."""
+        assert isinstance(future_market.offers, FutureOrders)
+        future_market.offers = {
+            str(offer.id): offer}
+        assert isinstance(future_market.offers, FutureOrders)
+        assert future_market.offers[str(offer.id)] == offer
+
```

Comment:
Lets add the check here, too. Or what do you think?
```suggestion
assert offer in offers.slot_order_mapping[offer.time_slot]

```

Reply:
And in the `test_bids_setter`, too.